retrofit/src/main/java/retrofit/Response.java

```diff
@@ -0,0 +1,63 @@
+package retrofit;
+
+import com.squareup.okhttp.Headers;
+import com.squareup.okhttp.ResponseBody;
+import java.io.IOException;
+import retrofit.converter.Converter;
+
+public final class Response<T> {
```

Comment:
This needs a little doc to differentiate _Result_ from _Response_ which are typically synonyms